TouchSky/TouchSky/TouchSky/Views/CompletedList.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CompletedList.swift
+//  TouchSky
+//
+//  Created by Alexa Nicole Hugenberg on 10/26/22.
+//
+
+import SwiftUI
+
+struct CompletedList: View {
+    @EnvironmentObject var modelData: ModelData
+    @State private var showCompletedOnly = true
+    
+    var filteredPeaks: [Peak] {
+        modelData.peaks.filter { peak in
+            (!showCompletedOnly || peak.isCompleted)
+        }
```

Comment:
Ok, so each of your `List` files is almost identical, with this little bit of logic being the main difference. I can see that you were trying to do something with `@State private var showCompletedOnly` and similar for ToDo, but it doesn't look like you ended up toggling those values ever?

So, really there are a couple ways to refactor this to share a single view. One would be that you have a `PeakList` view into which you pass some filtered values for each tab.

Another option would be to use a shared view that has some state variables like your `@State private var showCompletedOnly = true` and one for ToDo, and then that view knows how to dynamically filter and re-display the list.

Your list views are nice and clean because you have your views nicely separated, but you still don't want to have all these redundant extra views that you need to change any time you want to do something differently.